.chglog/CHANGELOG.tpl.md

```diff
@@ -33,10 +33,479 @@ A list of unreleased changes can be found [here]({{ .Info.RepositoryURL }}/compa
 {{ end -}}
 {{ end -}}
 
+## Overview
+
+### Breaking Changes
```

Comment:
I think this is the changelog from ui5-cli? 🤔

Reply:
Seems like we did this for rc.0 too, but no idea why: https://github.com/SAP/ui5-project/releases/tag/v3.0.0-rc.0